ksqldb-serde/src/main/java/io/confluent/ksql/serde/avro/AvroProperties.java

```diff
@@ -34,7 +34,8 @@ public class AvroProperties extends ConnectProperties {
 
   static final ImmutableSet<String> SUPPORTED_PROPERTIES = ImmutableSet.of(
       FULL_SCHEMA_NAME,
-      SCHEMA_ID
+      SCHEMA_ID,
+      SUBJECT_NAME
```

Comment:
Why do we need it for avro? I don't see it's used in avro serde

Reply:
Removed. I might use it later, but at least the regression issued is fixed.